GoogleForCreators/web-stories-wp
# Dashboard: Applied design updates to Entertainment template

## Summary

This PR applies the new design update to the Entertainment template.

## User-facing changes

Layout and text sizes have been updated on the Entertainment template based off the recommendations found in this [figma design doc](https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=766%3A61).

## Testing Instructions

1.) Go to "Explore Templates"
2.) Hover over the Entertainment template and click on "View".
3.) Take a look at each template page in the detail view for Entertainment. Make sure images are showing up and the layout appears as shown in the figma updates.

**NOTE:** `ent_page5_image1.jpg` has been updated but wordpress might cache the original image, so if on Page 5 if you don't see the new image depicted in the figma docs, clear the cache for this site and refresh and that should force the new image to load up.

---

Fixes #2474 

## Screenshots from Editor

<img src="https://user-images.githubusercontent.com/40646372/85061579-eaf0a080-b15b-11ea-8eda-d1f11f0933b0.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85061593-f3e17200-b15b-11ea-92cf-babac1fe803c.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85061622-ffcd3400-b15b-11ea-9b33-1cc49a52e4b8.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/85061644-08be0580-b15c-11ea-9960-7656ff36c146.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85061669-11aed700-b15c-11ea-90fb-2eca70f18211.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85061689-1a071200-b15c-11ea-861e-c8174f390a27.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/85061715-212e2000-b15c-11ea-8e76-68bbd1fc9c5a.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85061731-29865b00-b15c-11ea-8e3f-b154f7d45a22.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85061747-3145ff80-b15c-11ea-868c-1d86493e7dc5.png" height="200">

## Animations still work 🎉 

(If you set `enableAnimation` to `true` in `Dashboard.php`, you'll see that the animations still work under the new updates)

![ent_anim_still_work](https://user-images.githubusercontent.com/40646372/85062930-360bb300-b15e-11ea-8f6b-be4692176408.gif)
